src/RedemptionPool.sol

```diff
@@ -166,6 +186,22 @@ contract RedemptionPool is Ownable {
             revert RedemptionErrors.NoSweepGro();
         }
         // Transfers the tokens to the owner
-        IERC20(_token).safeTransfer(owner(), IERC20(_token).balanceOf(address(this)));
+        IERC20(_token).safeTransfer(
+            owner(),
+            IERC20(_token).balanceOf(address(this))
+        );
+    }
+
+    // @notice Transfers a portion or all of a user's redeemable GRO position to a new address.
+    // @param to The address to which the GRO position will be transferred.
+    // @param amount The amount of GRO to transfer.
+    function transferPosition(address to, uint256 amount) public {
+        if (amount > (_userGROBalance[msg.sender] - _userClaims[msg.sender]))
```

Comment:
Let them burn their positions into blackhole if they want to?

The DAO is winding down so not sure why they should be allowed to withdraw. The can transfer their position to another address if they want to move it?

Reply:
Because if contract contains bug, due to the speed and haste we are building it, there is an opportunity that something might be off and we might need to redeploy it.

With your setup it will mean we can sweep CUSDC but users will have the GRO tokens forever locked in previous bugged version

Reply:
I don't think there should be a sweep function for CUSDC either

Reply:
Well, then if it breaks - that's it. We are not going to have a proper audit on this, keep that in mind